src/metabase/lib/aggregation.cljc

```diff
@@ -206,3 +206,17 @@
        update :aggregation
        (fn [aggregations]
          (conj (vec aggregations) (lib.common/->op-arg query stage-number an-aggregate-clause)))))))
+
+(mu/defn aggregations :- [:maybe [:sequential lib.metadata/ColumnMetadata]]
```

Comment:
This stuff was already in `metabase.lib.stage`, but I thought this was a better place for it to live, since we'll probably want to use it elsewhere and `^:export` it at some point as well